src/shared/components/mutationTable/column/HgvscColumnFormatter.tsx

```diff
@@ -1,162 +1,63 @@
 import * as React from 'react';
-import { Circle } from 'better-react-spinkit';
 import 'rc-tooltip/assets/bootstrap_white.css';
 import { Mutation } from 'cbioportal-ts-api-client';
-import {
-    TableCellStatusIndicator,
-    TableCellStatus,
-} from 'cbioportal-frontend-commons';
+import { RemoteData } from 'cbioportal-utils';
 import { VariantAnnotation } from 'genome-nexus-ts-api-client';
-import GenomeNexusCache, {
-    GenomeNexusCacheDataType,
-} from 'shared/cache/GenomeNexusCache';
+import {
+    getHgvscColumnData,
+    Hgvsc,
+    hgvscDownload,
+    hgvscSortValue,
+} from 'react-mutation-mapper';
 
 export default class HgvscColumnFormatter {
     public static renderFunction(
         data: Mutation[],
-        genomeNexusCache: GenomeNexusCache | undefined,
+        indexedVariantAnnotations?: RemoteData<
+            { [genomicLocation: string]: VariantAnnotation } | undefined
+        >,
         selectedTranscriptId?: string
     ) {
-        const genomeNexusCacheData = HgvscColumnFormatter.getGenomeNexusDataFromCache(
```

Comment:
No need to cache additional genome nexus data. Everything we need is already in `indexedVariantAnnotations`